readthedocs/settings/base.py

```diff
@@ -610,6 +610,7 @@ def DOCKER_LIMITS(self):
     DEFAULT_VERSION_PRIVACY_LEVEL = 'public'
     GROK_API_HOST = 'https://api.grokthedocs.com'
     ALLOW_ADMIN = True
+    RTD_ALLOW_ORGANIZATIONS = False
```

Comment:
This setting will change to `RTD_ALLOW_ONLY_ORGANIZATONS` at some point when we support both single projects and organizations in .org :)